docs/webui.md

```diff
@@ -0,0 +1,206 @@
+---
+title: 'Gravitino webUI'
+slug: /webui
+keyword: webui
+toc_min_heading_level: 2
+toc_max_heading_level: 5
+license: 'Copyright 2023 Datastrato Pvt Ltd. This software is licensed under the Apache License version 2.'
+---
+
+> A graphical interface accessible through a web browser.
+
+Complete the installation, and open the browser [http://localhost:8090](http://localhost:8090).
+
+## Initial page
+
+The webUI homepage displaying in Gravitino depends on the configuration parameter for OAuth mode, see the details in [Security](security.md).
+
+Set parameter for `gravitino.authenticator`, [`simple`](#simple-mode) or [`oauth`](#oauth-mode).
+
+:::caution
+After the configuration change, make sure to restart the gravitino server.
+:::
+
+### Simple mode
+
+```
+gravitino.authenticator = simple
+```
+
+If the configuration parameter for `gravitino.authenticator` is set to `simple`, the webUI will directly display the homepage (Metalakes).
```

Comment:
Don't use the future tense, just using the simple tense: the web UI directly displays the homepage...

Reply:
> Don't use the future tense, just using the simple tense: the web UI directly displays the homepage...

DONE